pkgs/development/python-modules/zigpy-deconz/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, pyserial-asyncio, zigpy
```

this doesnt exist, did you meant pytest-asyncio?
```suggestion
, pyserial-asyncio, zigpy
```